python/lsst/analysis/drp/calcFunctors.py

```diff
@@ -132,43 +132,79 @@ def __call__(self, df):
 
 
 class CalcE(MultiColumnAction):
-    """Calculate a complex value representation of the ellipticity
-
-    This is a shape measurement used for doing QA on the ellipticity
-    of the sources.
+    """Calculate a complex value representation of the ellipticity.
 
     The complex ellipticity is typically defined as
-    E = ((ixx - iyy) + 1j*(2*ixy))/(ixx + iyy) = |E|exp(i*2*theta).
-
-    For plotting purposes we might want to plot |E|*exp(i*theta).
-    If `halvePhaseAngle` config parameter is set to `True`, then
-    the returned quantity therefore corresponds to |E|*exp(i*theta)
-    """
+    E = |E|exp(i*2*theta) = ((ixx - iyy) + 1j*(2*ixy))/(ixx + iyy).
+    This is sometimes referred to as distortion, and denoted by e = (e1, e2)
+    in GalSim and referred to as chi-type ellipticity following the notation
+    in Bartelmann and Schneider (2001). The other definition differs in the
+    denominator. It is referred to as shear, and denoted by g = (g1, g2) in
+    GalSim and referred to as epsilon-type ellipticity again following the
+    notation in Bartelmann and Schneider (2001). It is defined as
+    E = ((ixx - iyy) + 1j*(2*ixy))/(ixx + iyy + 2sqrt(ixx*iyy - ixy**2)).
 
-    colXx = Field(doc="The column name to get the xx shape component from.",
-                  dtype=str,
-                  default="ixx")
+    Notes
```

Comment:
Do you think it would be useful to add in the notes a short explanation of why we have two definitions, when we want to use one or the other, and why the chi-type is the default in the parameters below? Same comment for the two sizes introduced below.

Reply:
I added comments on the two types of definitions, but I'm going to refrain from commenting on the defaults. The defaults are left to whatever they were before this ticket is merged, so that the behavior of the functors called elsewhere remains unchanged. There's no other scientific rationale for the choice of these defaults.